packages/terra-aggregator/package.json

```diff
@@ -26,8 +26,9 @@
     "react-dom": "^16.8.5"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
```

Comment:
classnames is not used in terra-aggregator so I've removed it altogether from the package.json file.